src/App.js

```diff
@@ -142,15 +144,15 @@ class App extends React.Component<PropsT, StateT> {
     this.student = getVirtualStudent(this.studentName);
 
     this.studentImgId = Math.floor(Math.random() * numStudentImg);
-    this.studentLearningImg = `images/student/student_${
+    this.studentLearningImg = `/images/student/student_${
```

Comment:
I think this changes came up when you merged two branches. We should remove the first `/`, otherwise the url of images is wrong for the deployed app (although it works on localhost)